openeogeotrellis/deploy/batch_job.py

```diff
@@ -270,6 +270,7 @@ def run_job(
 ):
     result_metadata = {}
 
+    stac_hrefs = []
```

Comment:
If I understand correctly, this list is for collecting local paths. Using "href" in the name of this new variable is then not ideal as it  adds to the confusion between real hrefs and local paths as discussed in:
- https://github.com/Open-EO/openeo-geopyspark-driver/issues/926

Reply:
Is `stac_local_hrefs` a better name?
Href can be a s3 url, a https link, or a `file://paths` . 
At this point in the code it is still a filepath. 
After `_convert_job_metadatafile_outputs_to_s3_urls`, it becomes an s3 url.

Reply:
if that list contains local paths, I would avoid "href" and just call it `stac_paths`

I'm not that familiar with all the code in play here, but if those values are changed later to real hrefs, that's fine, but just switch to a different variable name only then